package.json

```diff
@@ -18,6 +18,7 @@
     ]
   },
   "scripts": {
+    "tsc:blade": "yarn workspace @razorpay/blade typecheck",
```

Comment:
why was the previous tsc running into issues? we already have eslint and tsconfig import alias resolvers defined at the root right? 🤔

Reply:
I have no idea why did we even had a root typecheck? Blade old is written in js what to typecheck?

Reply:
we used to generate types for it